content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,83 @@
+# check-ignore
+
+Used for debugging `.dvcignore` / exclude files.
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        Input files/directories to check ignore patterns.
```

Comment:
Why "inputs"?
files/directories -> paths?
Does it accept ignore patterns as targets?

Reply:
> Q though: Does it accept ignore patterns as targets?

I see it does from the example. So this should be mentioned in the `targets` short desc. I'm not committing this here yet because it affects the core repo. Will comment on that PR...

Reply:
From https://github.com/iterative/dvc/pull/4282#pullrequestreview-460270636:
```suggestion
  targets        File or directory paths, or ignore patterns to check
```

Reply:
@jorgeorpinel 

`or ignore patterns` is not correct most likely

Reply:
@shcheklein I think it's accurate. The argument help output is currently "Exact or wildcard paths of files or directories to check ignore patterns." (see https://github.com/iterative/dvc/pull/4282/files/930534ba85e83d34a45eead6e88a4722818af918#diff-655baccc374b563195716833da22b3baR64-R68)

Reply:
Committing this for now.